src/cryptography/hazmat/bindings/openssl/binding.py

```diff
@@ -91,14 +91,20 @@ def _ensure_ffi_initialized(cls):
         with cls._init_lock:
             if not cls._lib_loaded:
                 cls.lib = build_conditional_library(lib, CONDITIONAL_NAMES)
+                # init the OpenSSL locks
+                cls.init_static_locks()
```

Comment:
Sounds good.

On Sat, Aug 29, 2015 at 9:05 PM, Paul Kehrer <notifications@github.com>
wrote:

> In src/cryptography/hazmat/bindings/openssl/binding.py
> <https://github.com/pyca/cryptography/pull/2307#discussion_r38267563>:
>
> > @@ -91,14 +91,20 @@ def _ensure_ffi_initialized(cls):
> >          with cls._init_lock:
> >              if not cls._lib_loaded:
> >                  cls.lib = build_conditional_library(lib, CONDITIONAL_NAMES)
> > +                # init the OpenSSL locks
> > +                cls.init_static_locks()
>
> I'll move init_static_locks back out. Although I think we should do that
> too it can be in a separate PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2307/files#r38267563>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084